src/Chapter16.Tests/Listing16.06.Tests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter16.Listing16_06.Tests
+{
+    [TestClass]
+    public class ProgramTests
+    {
+        [TestMethod]
+        public void MainTest()
+        {
+            const string expected = @".\Microsoft.TestPlatform.AdapterUtilities.dll (2/15/2022 6:54:30 PM)
+.\Microsoft.TestPlatform.CommunicationUtilities.dll (5/10/2022 7:12:56 AM)
+.\Microsoft.TestPlatform.CoreUtilities.dll (5/10/2022 7:06:34 AM)
```

Comment:
This looks like a stack trace or something similar which is not what I think we want to be testing for as even if this works, right if there is a framework change anywhere in this stack it will fail

Reply:
mmm okay I understand